Automattic/jetpack
# PHPCS - Carousel audit

Brings carousel php files up to phpcs standards in order to remove them from phpcs ignore.

Changes so far are incomplete, but want to merge the mostly doc comment and small changes before breaking things with the bigger ones.

#### Changes proposed in this Pull Request:

* as above

#### Jetpack product discussion


n/a
#### Does this pull request change what data or activity we track or use?


n/a
#### Testing instructions:






* check out this branch
* enable the carousel
* make sure everything still works as expected
